src/Components/Login.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component }from 'react'
+import LoginForm from './LoginForm'
+import { Button, Message } from 'semantic-ui-react'
+import { signInUser } from '../state/actions/reduxTokenAuthConfig'
+import { connect } from 'react-redux'
+
+class Login extends Component {
+  state = {
+    renderLoginForm: false,
+    email: '',
+    password: '',
+    responseMessage: ''
+  }
+
+  renderFormState = () => {
+    this.setState({
+      renderLoginForm: !this.state.renderLoginForm
+    })
+  }
+
+  inputChangeHandler = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value
+    })
+  }
+  
+  handleLogin = () => {
+    const { signInUser } = this.props;
+    const { email, password } = this.state;
+      signInUser({ email, password })
+        .then(
+          console.log('Yaaaaaaay')
```

Comment:
Please make sure that the login form is not rendered when u have successfully been logged in.

Reply:
Fixed !